add draft site (#602)
digital-asset/docs.daml.com: docs/draft/README.md

announcement, even in the form of pre-alpha labs features.

Other than the completely separate ToC and contents, it functions in the same
way (same scripts to run, same editing workflow) as its sibling folders.

:arrow_right: If you encounter any issues, reach out to #product-docs on Slack.

## Live preview

Run the `live-preview` script to render a local view of the site.

```zsh
live-preview
```

The html files are in `workdir/build/gen`. Rerun the script to view any edits to `docs/` files or the top-level ToC (`_toc.yml`).

:warning: If you make changes to docs managed by the other repos, you have to commit them there. In this repository, you can only make changes to the top-level ToC (`_toc.yml`) or some section introduction pages in the `docs` folder.

:warning: If you do make a change to the TOCs in this repo, you then have to duplicate the change in the relevant satellite repo (except for the daml repo, which does not have a local ToC).

### Full build

To build the PDF docs or the exact HTML files that we publish to [docs.daml.com](https://docs.daml.com), run the full build locally:

```zsh
build
```

> The HTML files will be in `workdir/build/sphinx-target/html`; you can view them
> by doing something like:

Reviewer: ```suggestion
The HTML files are in `workdir/build/sphinx-target/html`; you can view them
by doing something like:
```

```zsh
$ cd workdir/build/sphinx-target/html
$ python -m http.server 8000 --bind 127.0.0.1
```

Then pointing your web browser to [http://localhost:8000](http://localhost:8000).

It also produces a PDF, and a tarball containing all of the HTML, under `workdir/target`.

## How to commit changes to this version of the docs

1. Make the changes to docs in the Daml, Daml Finance, or Canton repo, create a PR, and merge into main.

> :warning: You will have to set up and build the repo you are working on before making changes and viewing locally. Make sure you follow the build instructions carefully as they may differ to here.

Reviewer: ```suggestion
:warning: You must set up and build the repo you are working on before making changes and viewing locally. Make sure you follow the build instructions carefully as they may differ to here.
```
